pkg/util/request_validation.go

```diff
@@ -0,0 +1,33 @@
+package util
+
+import (
+	"errors"
+
+	admissionv1 "k8s.io/api/admission/v1"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
+
+// nolint: revive // Moved error out of pkg/webhook/admission; needs capitalization for backwards compat.
+var ErrOldObjectIsNil = errors.New("For admission webhooks registered for DELETE operations, please use Kubernetes v1.15.0+.")
+
+// RequireOldObjectOnDelete enforces that we use at least K8s API v1.15.0+ on DELETE operations
+// and copies over the oldObject into the Object field for the given AdmissionRequest.
+func RequireOldObjectOnDelete(req *admission.Request) error {
```

Comment:
This function name is slightly off, since calling it has a side effect.

`SetObjectOnDelete` ?